install_config/aggregate_logging.adoc

```diff
@@ -1343,6 +1343,206 @@ This causes Fluentd to write logs to the Fluentd default location, *_/var/log/fl
 oc set env ds/fluentd LOGGING_FILE_PATH=console
 ----
 
+[[fluentd-merge-json]]
+*Configuring log collection JSON parsing*
+
+You can configure the Fluentd log collector to determine if a log message is in *JSON* format and merge
+the message into the JSON payload document posted to Elasticsearch. This feature is disabled by default.
+
+You can enable or disable this feature by editing the `MERGE_JSON_LOG` environment variable in the *fluentd* daemonset.
+
+[IMPORTANT]
+====
+Enabling this feature comes with risks, including:
+
+* Possible log loss due to Elasticsearch rejecting documents due to inconsistent type mappings.
+* Potential buffer storage leak caused by rejected message cycling.
+* Overwrite of data for field with same names.
+
+The features in this topic should be used by only experienced Fluentd and Elasticsearch users.
```

Comment:
Omit this line, and say something like "If you suspect problems like this, or if you do not need to parse JSON from your logs, you should disable this feature."